### PR TITLE
README: get lftp from EPEL + promote libfaketime

### PR DESCRIPTION
**Pull request by ydirson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ XCP-ng versions.
 They require to install some additional packages first:
 
 ```
-sudo yum install -y genisoimage syslinux grub-tools createrepo_c ltfp
-sudo yum install -y --enablerepo=epel gnupg1 libfaketime
+sudo yum install -y genisoimage syslinux grub-tools createrepo_c
+sudo yum install -y --enablerepo=epel gnupg1 libfaketime ltfp
```
Comment (stormi) [inline README.md]: What's this new software, `ltfp`? :)

Comment (stormi) [reply README.md]: BTW, libfaketime is now in XCP-ng repos.

Comment (ydirson) [reply README.md]: Not that new indeed :D 
It's the tool I used for the partial repo-mirroring script.

Comment (stormi) [reply README.md]: I know `lftp`, but not `ltfp`.

Comment (stormi) [reply README.md]: Another comment: `lftp` is only required by the optional mirror-repos.sh script, so I would put it in a separate list. The reason we imported libfaketime is to make the ISO generation doable just with packages from the XCP-ng repos. Once `gnupg1` imported, we can reach this goal.

**Review by stormi — APPROVED** [review]

**Pull request merged**